app/cmd/root.go

```diff
@@ -93,6 +93,21 @@ func init() {
 	RootCmd.PersistentFlags().IntVar(&config.API.GrpcServerPort, "grpc-port",
 		config.API.GrpcServerPort, "GRPC api server port")
 
+	/**========================Hare Flags ========================== **/
+
+	// N determines the size of the hare committee
+	RootCmd.PersistentFlags().IntVar(&config.HARE.N, "hare-committee-size",
+		config.API.JSONServerPort, "Size of Hare committee")
+	// F determines the max number of adversaries in the Hare committee
+	RootCmd.PersistentFlags().IntVar(&config.HARE.F, "hare-max-adversaries",
+		config.API.JSONServerPort, "Max number of adversaries in the Hare committee")
+	// SetSize determines the size of the initial set
+	RootCmd.PersistentFlags().IntVar(&config.HARE.SetSize, "set-size",
```

Comment:
`set-size` is only used for initial map sizes. doesn't seem important enough for a config param right now. if this was approximate layer size (which corresponds to approx set-size) then it should be moved to a higher level config since it will be passed to more modules.

Reply:
Removed